### ISSUE TITLE
SafeConfigParser was removed in 3.12

### ISSUE THREAD

**Issue by tpw56j** [cobbler/cobbler#3551]
### Describe the bug

From https://docs.python.org/3.12/whatsnew/3.12.html:

Several names deprecated in the configparser way back in 3.2 have been removed per gh-89336:
- configparser.ParsingError no longer has a filename attribute or argument. Use the source attribute and argument instead.
- configparser no longer has a SafeConfigParser class. Use the shorter ConfigParser name instead.
- configparser.ConfigParser no longer has a readfp method. Use read_file() instead.

### Steps to reproduce

1. systemctl start cobblerd

### Cobbler version

````paste below
Cobbler 3.4.0
````

### Operating system

Fedora 39

### Cobbler log

````paste below
cobblerd[1614]: INFO | Exception raised when loading module authorization.configfile
cobblerd[1614]: INFO | Exception occurred: <class 'ImportError'>
cobblerd[1614]: INFO | Exception value: cannot import name 'SafeConfigParser' from 'configparser' (/usr/lib64/python3.12/configparser.py)
````

**Comment by SchoolGuy** [comment]
Yes I also just read this here: https://docs.python.org/3.6/whatsnew/3.2.html?highlight=safeconfigparser#configparser
